app/participants/knight.py

```diff
@@ -0,0 +1,23 @@
+class Knight:
+    def __init__(self, knight_config: dict) -> None:

```

Comment:
Implicit expectiations, when you create Knight object, you don't know what exactly should be in `knight_config`
To understand that I will need to go in this module and read the class logic. Change that